jax/tools/pgo_nsys_converter.py

```diff
@@ -0,0 +1,45 @@
+import csv
+import re
+import sys
+import argparse
+import psutil
+import os
+import shutil
+import subprocess
+
+nsys_path = shutil.which("nsys")
+
+parser = argparse.ArgumentParser(description='Tool to sweep for optimal collective combiner threshold')
```

Comment:
We can change the description to just nsys->pbtxt conversion, this doesnt do sweeping anymore.